app/views/contact/govuk/new.html.erb

```diff
@@ -12,38 +10,50 @@
     <% end %>
 
     <%= render partial: "shared/spam_honeypot", locals: { form_name: 'contact' } %>
+    <% if @errors %>
+    <%= render "govuk_publishing_components/components/error_summary", {
+      id: "error-summary",
+      title: "Please check the form",
+      # app adds a spam protection giraffe error, 
+      # which we do not want to display in the error summary
+      # so we filter it out
+      items: @errors.keys.reject { |key| key == :giraffe }.map do |key| 
+        {
+          text: @errors[key].first,
+          href: "##{key.to_s}error"
+        }
+      end
+    } %>
+    <% end %>
 
-    <%= render "shared/error_summary" %>
-
-    <fieldset id="location">
-      <legend>What's it to do with?</legend>
-        <label class="selectable">
-          <input type="radio" name="contact[location]" id="location-all" value="all"
-                 <%= @old ? (@old[:location] == "all" ? "checked" : "" ) : "checked" %> > The whole site
-        </label>
-
-      <% if @errors && @errors[:link] %>
-        <p class="validation group">
-        <span class="validation-message" id="linkerror"><%= @errors[:link].first %></span>
-      <% else %>
-        <p>
-      <% end %>
-
-      <label class="selectable">
-        <input type="radio" name="contact[location]" id="location-specific"
-         value="specific" <%= @old ? (@old[:location] == "specific" ? "checked" : "" ) : ""%> > A specific page
-      </label>
-
-        <label for="link" class="visuallyhidden">Enter URL or name of page</label>
-        <input type="text" name="contact[link]" id="link" placeholder="Enter URL or name of page"
-        value=<%= (@old ? @old[:link] : '') %> >
-        </p>
+    <fieldset class="govuk-fieldset govuk-!-margin-top-6">
+      <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">What's it to do with?</legend>
+      <div class="govuk-radios" data-module="govuk-radios">
+        <div class="gem-c-radio govuk-radios__item">
+          <input type="radio" name="contact[location]" id="location-all" value="all" class="govuk-radios__input" <%= @old ? (@old[:location] == "all" ? "checked" : "" ) : "checked" %> >
```

Comment:
Both of the radio inputs have the `checked` attribute when I visit `/contact/govuk/` directly - is that the desired behaviour?

![Screen Shot 2019-09-03 at 11 42 02](https://user-images.githubusercontent.com/1732331/64166869-e57dd000-ce3f-11e9-9fec-34ccaf7396bc.png)

Reply:
probably not, looking at the checked logic which hasn't changed 🤔